test/extended/util/annotate/rules.go

```diff
@@ -113,6 +113,11 @@ var (
 		"[Flaky]": {
 			`Job should run a job to completion when tasks sometimes fail and are not locally restarted`, // seems flaky, also may require too many resources
 			`openshift mongodb replication creating from a template`,                                     // flaking on deployment
+			// Skip the following 4 tests until https://bugzilla.redhat.com/show_bug.cgi?id=1794714 is fixed.
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1822714

Reply:
I updated the inline doc to reference [BZ 1822714](https://bugzilla.redhat.com/show_bug.cgi?id=1822714).